pkgs/tools/package-management/apx/default.nix

```diff
@@ -30,25 +30,24 @@ buildGoModule rec {
   postInstall = ''
     mkdir -p $out/etc/apx
 
-    cat > "$out/etc/apx/config.json" <<EOF
+    # https://github.com/Vanilla-OS/apx/blob/main/config/apx.json
+    cat > "$out/etc/apx/apx.json" <<EOF
     {
-      "containername": "apx_managed",
-      "image": "docker.io/library/ubuntu",
-      "pkgmanager": "apt",
-      "distroboxpath": "${distrobox}/bin/distrobox"
+      "apxPath": "$out/bin/apx",
+      "distroboxpath": "${distrobox}/bin/distrobox",
+      "storageDriver": "btrfs"
     }
     EOF
 
-    wrapProgram $out/bin/apx --prefix PATH : ${lib.makeBinPath [ docker distrobox ]}
-
-    installManPage man/de/man1/apx.1 man/es/man1/apx.1 man/fr/man1/apx.1 man/it/man1/apx.1 man/man1/apx.1 man/nl/man1/apx.1 man/pl/man1/apx.1 man/pt/man1/apx.1 man/pt_BR/man1/apx.1 man/ro/man1/apx.1 man/ru/man1/apx.1 man/sv/man1/apx.1 man/tr/man1/apx.1
+    wrapProgram $out/bin/apx --prefix PATH : ${lib.makeBinPath [ podman distrobox ]}
```

Comment:
There is no need to wrap it with distrobox, since it's path is already hardcoded in the config file.